docs/docs-content/clusters/edge/trusted-boot/deployment-day2/install.md

```diff
@@ -0,0 +1,164 @@
+included in this guide involve the Basic Input/Output System (BIOS) interface of the Edge device which varies from
+machine to machine. This guide uses the Intel NUC 13 Pro as an example.
+
+## Prerequisites
+
+- Edge Installer ISO file. Check out the [Build Edge Artifacts with Trusted Boot](../edgeforge/build-trusted-iso.md) to
+  learn how to create an Edge Installer ISO with Trusted Boot.
+- The ability to modify the boot order settings to boot from a USB drive and to enter the BIOS interface. Typically this
+  requires you to have a wired keyboard and a monitor to display the interface connected to the Edge device.
+- A USB disk containing the installer ISO.
+
+:::warning
+
+Before proceeding with installation, check the maximum EFI size that your Edge host is able to boot and compare it
+against the installer ISO and provider image. With secure boot enabled and keys enrolled, you won't be able to boot from
+the EFI size checking ISO unless you disable secure boot or clear the keys. For more information, refer to
+[Check EFI Size and Edge Host Boot Limit](../edgeforge/check-efi-limit.md).
+
+:::
+
+## Instructions
+
+### Enroll Secure Boot Keys into Edge Device
+
+The following steps are written for the Intel NUC 13 Pro device. If you are using a different device, the exact sequence
+of steps might be different. However, you should be able to find the same or similarly worded options in any device that
+supports Secure Boot.
+
+1. Insert the USB disk into the Edge device.
+
+2. Power up your Edge device and enter the BIOS interface. Oftentimes you can accomplish this by pressing F2, F1, or F10
+   on the keyboard immediately after powering up the deice and before you select a boot volume, as is the case in Intel
+   NUC 13 Pro. However, the exact method might be different on your Edge device. Consult the manufacturer of your Edge
+   device to find out how to enter the BIOS interface.
+
+3. From the BIOS menu, select the **Boot** tab. Then select **Secure Boot**.
+
+4. From the **Secure Boot Menu**, make sure that secure boot is enabled. Then select **Restore to Setup Mode**. Two
+   dialogue boxes will pop up to confirm this operation. Select **Yes** to confirm. This will make the device restart.
+
+   :::info
+
+   **User Mode** and **Setup Mode** are UEFI terms. Setting secure boot mode to **Setup Mode** means clearing the keys
+   that are currently in the device. This allows you to enroll your own keys, while **User Mode** means you have
+   enrolled your own keys.
+
+   In some BIOS, the same actions or functionality might be named differently in the BIOS interface. Consult the
+   manufacturer of your Edge host to find out how to clear the keys and enroll your own keys.
+
+   :::
+
+If you set the `AUTO_ENROLL_SECUREBOOT_KEYS` argument to `true` when you built the installation ISO, key enrollment will
+begin automatically once the device restarts. You can skip the following steps to proceed to
```

Comment:
```suggestion
begin automatically the first time the device boots from the installer ISO. You can skip the following steps to proceed to
```

Reply:
Note: this option will be disregarded in VM. So in VMs, keys will always be auto enrolled.